src/dsl/verifier.ts

```diff
@@ -156,6 +157,7 @@ export class Verifier {
     app.all("/*", (req, res) => {
       logger.debug("Proxing", req.path)
       proxy.web(req, res, {
+        changeOrigin: this.config.changeOrigin === false,
```

Comment:
Is this correct? Doesn't this mean if you set `changeOrigin` to `false` it would set it to `true`?

Reply:
Will pull in the project on this branch now and just test this out, you are probably correct , apologies!

Reply:
No worries. I think you could copy the check for `secure`.

Reply:
Cheers, just checked it out and it was set the wrong way around. Push up the change now.

I can't get my head around why the need for the `=== true` check.

I get the same behaviour without the check, if I pass in true, false as booleans, or don't provide, as lines 214-216 will set it to false if undefined.

If I just amend the js file to avoid typescript checking as pass in a string  in opts such as `  changeOrigin: "cheese",` rather than the boolean specified in the typescript interface, http-proxy seems to accept the fact that changeOrigin is provided irrespective of value.